backend/Dockerfile-dev

```diff
@@ -7,4 +7,6 @@ WORKDIR /app
 RUN pip install -r requirements.txt
 
 ENV FLASK_ENV=docker
-EXPOSE 5000
\ No newline at end of file
+EXPOSE 5000
+
+CMD ["python", "manage.py", "runserver"]
```

Comment:
So Dockerfile-dev is used in `docker-compose` but if you aren't using that, it's fine